test/ImpactEvaluator.t.sol

```diff
@@ -230,6 +230,30 @@ contract ImpactEvaluatorTest is Test {
         impactEvaluator.setScores(0, addresses, scores);
     }
 
+    function test_SetScoresAfterUpdateMinBalanceForTransfer() public {
+        ImpactEvaluator impactEvaluator = new ImpactEvaluator(address(this));
+        vm.deal(payable(address(impactEvaluator)), 100 ether);
+
+        impactEvaluator.adminAdvanceRound();
+        impactEvaluator.adminAdvanceRound();
+
+        address payable[] memory addresses = new address payable[](2);
+        addresses[0] = payable(vm.addr(1));
+        addresses[1] = payable(0x000000000000000000000000000000000000dEaD);
+        uint[] memory scores = new uint[](2);
+        scores[0] = 1;
+        scores[1] = impactEvaluator.MAX_SCORE() - scores[0];
+
+        impactEvaluator.setScores(1, addresses, scores);
+        assert(!impactEvaluator.participantIsReadyForTransfer(addresses[0]));
+
+        impactEvaluator.setMinBalanceForTransfer(0);
+        impactEvaluator.adminAdvanceRound();
+
+        impactEvaluator.setScores(2, addresses, scores);
```

Comment:
Updated test in https://github.com/Meridian-IE/impact-evaluator/pull/81/commits/94fc92f867ec9844619ee5654ddcaaab40b2e5cc

> I think this is fine as long as we can manually call setScores with 0 score for each known participant address. Is that how you envision this will be used?

I didn't think about manually setting their scores to 0, to me it would be that the next time they commit a measurement they will be included in setScores(), and thereby potentially marked for transfer. In case they're not part of the network any more, this doesn't work, so yeah in order to catch everyone we would need to submit 0 for all known participants that aren't included in the first round after the threshold change.